CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - (Keep your changes here until you have a release version)
 
+## [1.1.2] - 2024-05-31
```

Comment:
Suggest updating package.json to this version and running "npm install" so it updates the lock file too.  MS Word Compare Documents feature worked great for seeing those changes.  Will hold off on approval until after hearing back about the links so you don't have to do a new PR if he did want different links.